tests/fixtures/locator.spdx.json

```diff
@@ -0,0 +1,162 @@
+{
+    "spdxVersion": "SPDX-2.2",
+    "documentNamespace": "https://localmaven",
+    "creationInfo": {
+        "creators": [
+            "Tool: github.com/phylum-dev/cli"
+        ],
+        "created": "2023-04-04T20:15:57Z",
+        "licenseListVersion": "3.20",
+        "comment": "SBOM Document for the localmaven project from Phylum, Inc."
+    },
+    "dataLicense": "CC0-1.0",
+    "SPDXID": "SPDXRef-DOCUMENT",
+    "name": "localmaven-SBOM",
+    "packages": [
+        {
+            "SPDXID": "SPDXRef-org.jruby:jruby-complete-9.3.7.0",
+            "name": "org.jruby:jruby-complete",
+            "downloadLocation": "https://repo1.maven.org/maven2/org/jruby/jruby-complete/9.3.7.0/jruby-complete-9.3.7.0-sources.jar",
+            "filesAnalyzed": true,
+            "licenseConcluded": "NOASSERTION",
+            "copyrightText": "NOASSERTION",
+            "versionInfo": "9.3.7.0",
+            "supplier": "NOASSERTION",
+            "homepage": "https://central.sonatype.com/artifact/org.jruby/jruby-complete/9.3.7.0",
+            "externalRefs": [
+                {
+                    "referenceCategory": "PACKAGE-MANAGER",
+                    "referenceType": "maven",
+                    "referenceLocator": "org.jruby:jruby-complete@9.3.7.0"
+                }
+            ]
```

Comment:
Looking at the [spec][1], I think this should be `"referenceType": "maven-central"`...

[1]: https://spdx.github.io/spdx-spec/v2.3/external-repository-identifiers/#f31-maven-central

Reply:
I just realized that the `"referenceLocator"` doesn't match the spec either...

Accoding to the spec, this should be `"referenceLocator": "org.jruby:jruby-complete:9.3.7.0"`